src/LandIce/problems/LandIce_StokesFOThermoCoupled.hpp

```diff
@@ -405,7 +382,7 @@ constructEnthalpyEvaluators (PHX::FieldManager<PHAL::AlbanyTraits>& fm0,
  *   p->set<std::string>("Gradient BF Side Name", Albany::grad_bf_name + " "+basalSideName);
  *   p->set<std::string>("Weighted Measure Name", Albany::weighted_measure_name + " "+basalSideName);
  *   p->set<std::string>("Velocity Side QP Variable Name", dof_names[0] + "_" + basalSideName);
- *   p->set<std::string>("Vertical Velocity Side QP Variable Name", "W");
+ *   p->set<std::string>("Vertical Velocity Side QP Variable Name", dof_names[0]);
```

Comment:
`dof_names[0]` should be `Velocity'. Is this a bug? Shouldn't it maybe be `dof_names[1]`? If 'Velocity' is correct, maybe the param name should not be `Vertical Velocity ...`, but `Velocity ...`.

Reply:
you are right. I'll fix it.

Reply:
fixed.